test/conformance/gateway_conformance_test.go

```diff
@@ -41,7 +44,9 @@ var expressionRoutesSupportedFeatures = []suite.SupportedFeature{
 	suite.SupportHTTPRouteQueryParamMatching,
 	suite.SupportHTTPRouteMethodMatching,
 	suite.SupportHTTPRouteResponseHeaderModification,
-	suite.SupportHTTPRouteBackendTimeout,
+	// Temporarily disabled and tracking through the following issue.
+	// https://github.com/Kong/kubernetes-ingress-controller/issues/5868
```

Comment:
Just to be sure this is a todo not a solved issue. I believe we had a check somewhere to verify issues having `TODO: <issue_link>`  that are closed.

https://github.com/Kong/kubernetes-ingress-controller/blob/cefe68030ca9cc2a0b02b140fac1c2129f040bf4/.github/workflows/check_fixed_issues_references.yaml#L17

Reply:
Thanks!